cmd/ethrex/initializers.rs

```
@@ -391,6 +391,10 @@ pub async fn init_l1(
 
     let blockchain = init_blockchain(store.clone(), BlockchainType::L1, true);
 
+    MempoolPruner::spawn(blockchain.clone())
+        .await
+        .expect("Failed to start mempool pruner");
```

Shouldn't we propagate this error?